DeepSpeech.ipynb

```diff
@@ -446,7 +446,7 @@
     "    lstm_bw_cell = tf.nn.rnn_cell.BasicLSTMCell(n_cell_dim, forget_bias=1.0)\n",
     "    \n",
     "    # Split data because rnn cell needs a list of inputs for the BRNN inner loop\n",
-    "    layer_3 = tf.split(0, n_steps, layer_3)\n",
```

Comment:
Could be wrong. I'm too jet lagged to think. Can you verify and suggest an alternative.